manifests/charts/base/values.yaml

```diff
@@ -36,3 +31,8 @@ defaults:
     enableIstioConfigCRDs: true
 
   defaultRevision: "default"
+
+# Platform where Istio is deployed. Possible values are: "openshift", "gcp".
+# An empty value means it is a vanilla Kubernetes distribution, therefore no special
+# treatment will be considered.
+platform: ""
```

Comment:
I think this should also be placed under `defaults` to maintain consistency with the merge logic.